common/server.conf

```diff
@@ -1,6 +1,3 @@
-ssl_certificate common/snakeoil.crt;
```

Comment:
The reason for this change is two-fold:
* AR by default listens on 0.0.0.0:80, this makes it impossible to mock out  i.e. marathon leader proxying
* all listen and ssl_* related stuff in EE is in a separate file as well. Unifying the approach will make the merge a bit easier.